src/Explorer/Controls/InputTypeahead/InputTypeahead.ts

```diff
@@ -106,6 +106,8 @@ class InputTypeaheadViewModel {
     this.params = params;
 
     this.params.choices.subscribe(this.initializeTypeahead.bind(this));
+    this.inputValue = "";
+    this.selection = null as any;

```

Comment:
We should either remove the private members since they are not being used in HTML bindings either

Reply:
I remove them